doc/substitutions.yaml

```diff
@@ -1,2 +1,2 @@
 # Key/value substitutions to use within the Sphinx doc.
-{}
+{note_ip_addresses_CIDR: "IP addresses and subnets must be specified in CIDR notation (for example, `192.0.2.0/24` or `2001:db8::/32`)."}
```

Comment:
I think we do need it on a per-option basis because it's not always true.

For example `ipv4.address` on a NIC device isn't CIDR but entries in `ipv4.routes` or `ipv4.routes.external` are CIDR.
On a network, `ipv4.address` and `ipv4.gateway` are CIDR but `ipv4.nat.address` is not.